README.md

```diff
@@ -244,3 +244,7 @@ This provides you with a few different routes:
 * Proper metadata mapping is necessary, allow to disable versioning for fields and associations.
 * It does NOT work with Joined-Table-Inheritance (Single Table Inheritance should work, but not tested)
 * Many-To-Many associations are NOT versioned
+
+## Contributing
+
+Before each commit please run the `make cs` command.
```

Comment:
Could you remove the `Makefile` and refer those commands in the `README.md` instead?